lib/plugins/aws/custom-resources/index.js

```diff
@@ -162,6 +162,12 @@ async function addCustomResourceToService(awsProvider, resourceName, iamRoleStat
     });
   }
 
+  let runtimeVersion = 'nodejs18.x';
+  const providerRuntime = awsProvider.getRuntime();
+  if (providerRuntime.indexOf('nodejs') !== -1) {
```

Comment:
Won't this be more easy to read with `startsWith` instead?
```suggestion
  if (providerRuntime.startsWith('nodejs')) {
```

Reply:
It does, but it retuned an error somehow when using that and running the tests against it

Reply:
Updated the code, ran the tests again successfully.